.hlint.yaml

```diff
@@ -0,0 +1,93 @@
+###########################################################################
+# Settings
+###########################################################################
+
+- arguments: [-XTypeApplications, -XRecursiveDo]
+
+# These are just too annoying
+- ignore: { name: Redundant do }
+- ignore: { name: Redundant bracket }
+- ignore: { name: Redundant lambda }
+- ignore: { name: Redundant $ }
+- ignore: { name: Redundant flip }
+- ignore: { name: Move brackets to avoid $ }
+
+# Losing variable names can be not-nice
+- ignore: { name: Eta reduce }
+- ignore: { name: Avoid lambda }
+
+# Humans know better
+- ignore: { name: Use camelCase }
+- ignore: { name: Use const }
+- ignore: { name: Use section }
+- ignore: { name: Use if }
+- ignore: { name: Use notElem }
+- ignore: { name: Use fromMaybe }
+- ignore: { name: Use maybe }
+- ignore: { name: Use fmap }
+- ignore: { name: Use foldl }
+- ignore: { name: "Use :" }
+- ignore: { name: Use ++ }
+- ignore: { name: Use || }
+- ignore: { name: Use && }
+- ignore: { name: 'Use ?~' }
+- ignore: { name: Use <$> }
+
+# There's no 'head' in Universum
+- ignore: { name: Use head }
+
+# Sometimes [Char] is okay (if it means "a set of characters")
+- ignore: { name: Use String }
+
+# We have 'whenJust' for this
+- ignore: { name: Use Foldable.forM_ }
+
+# Sometimes TemplateHaskell is needed to please stylish-haskell
+- ignore: { name: Unused LANGUAGE pragma }
+
+# Some 'data' records will be extended with more fields later,
+# so they shouldn't be replaced with 'newtype' blindly
+- ignore: { name: Use newtype instead of data }
+
+
+###########################################################################
+# Hints with 'id' should use 'identity'
```

Comment:
`universum`-specific and doesn't even make any sense nowadays, because `identity` is deprecated.